src/rust/bitbox02-rust/src/shiftcrypto.bitbox02.rs

```diff
@@ -1,103 +1,99 @@
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct PubResponse {
-    #[prost(string, tag="1")]
+    #[prost(string, tag = "1")]
```

Comment:
When I build locally, this files gets reformatted. Did your editor auto-formatted it on save?

Reply:
somehow i get this reformatting. need to figure out why. will fix

Reply:
done